index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Conference</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/a076d05399.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+
+  <body>
+    <header>
+      <div class="header-nav">
+        <ul class="icons">
+          <li class="icon">
+            <a href="#"> <i class="fab fa-facebook-f"></i></a>
+          </li>
+          <li class="icon">
+            <a href="#">
+              <i class="fab fa-twitter"></i>
+            </a>
+          </li>
+          <li class="icon">
+            <a href="#" class="none">English </a>
+          </li>
+          <li class="icon">
+            <a href="#" class="none">My Page </a>
+          </li>
+          <li class="icon">
+            <a href="#" class="none">Logout </a>
+          </li>
+        </ul>
+      </div>
+
+      <nav class="nav-bar">
+        <a href="index.html" class="logo">Math Events</a>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <div class="nav-menu">
+          <ul class="nav-links">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#" class="nav-link">Program</a></li>
+            <li><a href="#" class="nav-link">Join</a></li>
+            <li><a href="#" class="nav-link">Sponsor</a></li>
+            <li><a href="#" class="news nav-link">News</a></li>
+          </ul>
+          <button class="math" href="#">Math Events</button>
+        </div>
+      </nav>
+    </header>
+    <div class="welcome">
+      <div class="container">
+        <h1>"Hello!Sharing world"</h1>
+        <h2>Workshop on Operator Algebras and Applications 2023</h2>
+        <div class="first">
+          <p>
+            A joyful celebration believing in the value of openness and sharing,
+            creating a positive change with people from over 80 countries is
+            taking place in September, in Canada.
+          </p>
+        </div>
+        <h3>2023.09.01(FRI) ~ 3(SAT)</h3>
+        <h4>@ National Museum of Karea, Art Center Nabi and more</h4>
+      </div>
+    </div>
+    <div class="program">
+      <div class="container2">
+        <h1>Main Program</h1>
```

Comment:
- [ ] 🙏 Please avoid using more than one `<h1>` heading on the same page for the purpose of search engine optimization. Search engines consider the `<h1>` heading as a significant element for determining the main topic or focus of a webpage🖥. By limiting the usage of `<h1>` to a single instance, you ensure👍 that search engines accurately understand the primary subject matter of your page📜, which can positively impact its ranking and visibility🙄 in search results.